frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -155,21 +155,21 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
             LOG.info("Processing request");
 
             var session = userContext.getSession();
-            Optional<AuthSessionItem> authSession =
+            Optional<AuthSessionItem> authSessionOptional =
                     authSessionService.getSessionFromRequestHeaders(input.getHeaders());
-            if (authSession.isEmpty()) {
+            if (authSessionOptional.isEmpty()) {
```

Comment:
It might be good to assign the optional to a variable rather than calling get throughout the handler

Reply:
Yes, that'd be better and is more consistent with what's done in the verify mfa code handler